test/client/fixtures/utils/style-test.js

```diff
@@ -187,3 +188,46 @@ test('getOffset', function () {
     document.body.style.border            = '';
     document.documentElement.style.border = '';
 });
+
+test('getSelectElementSize', function () {
+    function createOption (parent, text) {
+        return $('<option></option>').text(text)
```

Comment:
@churkin It's not a big deal. This auto-generated by editor auto-completion, so it's ok to keep it as is.